app/Exceptions/Handler.php

```diff
@@ -120,7 +120,7 @@ public function render($request, Exception $e)
 
     protected function unauthenticated($request, AuthenticationException $exception)
     {
-        if ($request->expectsJson()) {
+        if ($request->wantsJson()) {
```

Comment:
seems fine to use `expectsJson` here?

![](https://puu.sh/AvFDv/8972e10768.png)

https://github.com/laravel/framework/blob/2906b572aa83ff46b8ad57093dd0e859c0ff783f/src/Illuminate/Http/Request.php#L704